R/agg_scale.R

```diff
@@ -83,6 +88,15 @@
 #' location-years have 5-year age groups while other location-years have 1-year
 #' age groups.
 #'
+#' In addition it is okay if the interval variables included in `id_vars` are
```

Comment:
I'd also clarify that it's not that id _variables_ can be different, it's that the unique _values_ of those variables can be different so long as aggregation to a common set is possible.

Reply:
What happens if aggregation to a common set is not possible? Example: [0-15, 15-60, 60+] & [0-10, 10-20, ... , 100+].

Reply:
There actually is a common set there, 0-Inf. But I'm not sure how the function behaves if its something like [0-10, 10-20], & [20-30, 40+]. Not sure how much time its worth spending on this issue